icu4c/source/i18n/decimfmt.cpp

```diff
@@ -1609,8 +1613,9 @@ void DecimalFormat::touch(UErrorCode& status) {
     }
 
     // In C++, fields->symbols is the source of truth for the locale.
```

Comment:
Just curious, is the comment about `fields->symbols` being the source of truth still accurate?

Reply:
Not really.  I went back and tried to wordsmith this comment so that it's true again.

Reply:
Thanks!